### ISSUE TITLE
Ambiguity of label predance

### ISSUE THREAD

**Issue by SeniorMars** [typst/typst#449]
Hello, I just finished correctly parsing typst's code mode through treesitter: https://github.com/SeniorMars/tree-sitter-typst. however, I've noticed that in one of your examples, you have the following:

```
#set text(blue) if (it.has("label") and it.label) == <label>
#set text(blue) if it.has("label") and (it.label == <label>)
```

I searched your website, but I need help how interpreting this correctly. From my gut feeling, it is number 2, but I want to be sure.

Also, it would be nice to know whether label counts officially as an "expression" in your language spec.

**Comment by SeniorMars** [comment]
Thank you! That's what I thought. Also, typst is very hard to parse. Unicode and whitespace do not make it easy 😅

**Comment by laurmaedje** [comment]
True! I'm gonna close this then.
